plugins/cassandra/cassandrawu.cpp

```diff
@@ -3798,3 +3802,13 @@ extern "C" EXPORT IWorkUnitFactory *createWorkUnitFactory(const IPropertyTree *p
 {
     return new cassandraembed::CCasssandraWorkUnitFactory(props);
 }
+
+/*
+ * Graph progress thoughts:
+ * 1. graph state gets 'merged in' to the meta (lightweight) iterator results and is therefore available in the heavy ones too.
+ * 2. It's used via both interfaces, but only in the esp cases (both of which iterate in fill).
```

Comment:
I think I'm inclined to delete the comment altogether, since it refers to what the code was like BEFORE I made my changes rather than after.